packages/neuron-wallet/src/services/skip-data-and-type.ts

```diff
@@ -0,0 +1,51 @@
+import FileService from './file'
+
+export default class SkipDataAndType {
+  private static moduleName = ''
+  private static fileName = 'skip-data-and-type.json'
```

Comment:
How about a global `settings.json` for all related persisted app configuration? I don't see any benefit maintaining a single file for each feature.

Reply:
Add a `BaseSettings` to read/update settings and read/update `SkipDataAndType` via this.